src/Model/Shipment.php

```diff
@@ -46,6 +46,13 @@ class Shipment
      */
     public $shippingProductId = 0;
 
+    /**
+     * 0 if Shipment, 1 if Retoure
+     * @var int
+     * @DataField(name="ShipmentType", type="int")
+     */
+    public $shipmentType = 0;
```

Comment:
Instead of using magic numbers it's better to use constants. Please create a `src/Type/ShipmentType.php` and add the constants SHIPMENT=0 and RETOURE=1. Use ShipmentType::SHIPMENT in this class.
 (Similar to the definition of [ProductCondition](https://github.com/billbeeio/billbee-php-sdk/blob/master/src/Type/ProductCondition.php) and its usage: https://github.com/billbeeio/billbee-php-sdk/blob/07409bccd74cc91b9f053931a0de246c5c27892f/src/Model/Product.php#L416)